docs/src/changelog.md

```diff
@@ -7,6 +7,30 @@ CurrentModule = JuMP
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Version 1.9.0 (March 1, 2023)
+
+### Added
+
+ - Added [`get_attribute`](@ref) and [`set_attribute`](@ref). These replace
+   [`get_optimizer_attribute`](@ref) and [`set_optimizer_attribute`](@ref),
+   although the `_optimizer_` functions remain to avoid breakage. (#3219)
```

Comment:
```suggestion
   although the `_optimizer_` functions remain to avoid breakage. (#3219)
 - Added support for `Base.isreal` for variables and expressions (#3252)
```